refinery/file_store/utils.py

```diff
@@ -158,9 +156,9 @@ def get_file_size(file_location):
             return UNKNOWN_FILE_SIZE
     else:
         try:
-            with contextlib.closing(urllib2.urlopen(file_location,
-                                                    timeout=30)) as response:
-                return int(response.info().getheader('Content-Length'))
+            with contextlib.closing(urllib.request.urlopen(file_location,
+                                    timeout=30)) as response:
+                return int(response.info().get('Content-Length'))
```

Comment:
Should this be: `return int(response.getheader('Content-Length'))`?
https://docs.python.org/3/library/http.client.html#http.client.HTTPResponse.getheader